### PR TITLE
Placeholder on Criteria Editor title

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

**Comment by andresfulla** [comment]
Hi @bryceosterhaus ,

Just some bits to include a placeholder in the Criteria Editor title.

This one is on top of https://github.com/bryceosterhaus/liferay-portal/pull/52

Thanks!

**Comment by andresfulla** [comment]
@dgarciasarai

**Comment by dgarciasarai** [comment]
Commit message: "LPS-96508 ~placeholder~ localized input now supports placeholder property"

File: modules/apps/segments/segments-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -62,6 +62,7 @@ no-users-have-been-assigned-to-this-segment=No users have been assigned to this
 not-contains=Not Contains
 not-equals=Not Equals
 preview-members=Preview Members
+segment-name=Segment Name
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-lang/src/main/resources/content/Language.properties]: the placeholder should be "Untitled Segment" to be more consistent with other parts of the portal? for example, Web Content? (Untitled Basic Web Content)

also, the lang key `unnamed-segment=Unnamed Segment` is not in use anymore, so delete, please :)

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js [hunk 2]
```diff
@@ -338,6 +338,8 @@ class SegmentEdit extends Component {
 			values
 		} = this.props;
 
+		const criteriaNamePlaceholder = Liferay.Language.get('segment-name');
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js]: why not `placeholder`?

File: modules/apps/segments/segments-web/test/js/components/segment_edit/__snapshots__/SegmentEdit.es.js.snap [hunk 3]
```diff
@@ -78,7 +78,7 @@ exports[`SegmentEdit should render 1`] = `
               <input
                 class="rounded form-control language-value field form-control-inline form-control"
                 data-testid="localized-main-input"
-                placeholder=""
+                placeholder="segment-name"
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-web/test/js/components/segment_edit/__snapshots__/SegmentEdit.es.js.snap]: if you change the lang key, be careful and rebuild lang

**Comment by dgarciasarai** [comment]
@bryceosterhaus I'm going to change these little things and I'll resend to you :) thanks!

**Comment by andresfulla** [comment]
Closing. @dgarciasarai is resending 🙏

**Comment by dgarciasarai** [comment]
Resent here: https://github.com/bryceosterhaus/liferay-portal/pull/54

**Pull request closed without merging**